diff --git a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/EnemyGame.java b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/EnemyGame.java
index 7ad219205..9487d57d4 100644
--- a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/EnemyGame.java
+++ b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/EnemyGame.java
@@ -18,7 +18,7 @@ import com.betsoft.casino.mp.service.ISpawnConfigProvider;
 import com.betsoft.casino.mp.service.ITransportObjectsFactoryService;
 import com.betsoft.casino.teststand.TestStandFeature;
 import com.betsoft.casino.teststand.TestStandLocal;
-import com.dgphoenix.casino.common.exception.CommonException;
+import com.abs.casino.common.exception.CommonException;
 import com.abs.casino.common.util.RNG;
 import org.apache.logging.log4j.Logger;
 import java.util.*;
diff --git a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/GameMap.java b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/GameMap.java
index 3f9e8dcfd..cebab27ab 100644
--- a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/GameMap.java
+++ b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/GameMap.java
@@ -10,7 +10,7 @@ import com.betsoft.casino.mp.common.scenarios.TypeAndSkin;
 import com.betsoft.casino.mp.model.*;
 import com.betsoft.casino.mp.model.movement.*;
 import com.betsoft.casino.mp.movement.SpiderSwarmAngleTrajectoryGenerator;
-import com.dgphoenix.casino.common.util.Pair;
+import com.abs.casino.common.util.Pair;
 import com.abs.casino.common.util.RNG;
 import com.esotericsoftware.kryo.Kryo;
 import com.esotericsoftware.kryo.io.Input;
diff --git a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/GameRoom.java b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/GameRoom.java
index 340796b54..e4ec1ab89 100644
--- a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/GameRoom.java
+++ b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/GameRoom.java
@@ -11,7 +11,7 @@ import com.betsoft.casino.mp.model.movement.Trajectory;
 import com.betsoft.casino.mp.model.room.ISingleNodeRoomInfo;
 import com.betsoft.casino.mp.service.*;
 import com.betsoft.casino.mp.web.IGameSocketClient;
-import com.dgphoenix.casino.common.exception.CommonException;
+import com.abs.casino.common.exception.CommonException;
 import com.hazelcast.core.IExecutorService;
 import org.apache.logging.log4j.Logger;
 import org.springframework.context.ApplicationContext;
diff --git a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/PlayGameState.java b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/PlayGameState.java
index 63882de10..355a05c4a 100644
--- a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/PlayGameState.java
+++ b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/PlayGameState.java
@@ -13,8 +13,8 @@ import com.betsoft.casino.teststand.TestStandLocal;
 import com.betsoft.casino.teststand.TeststandConst;
 import com.betsoft.casino.utils.TInboundObject;
 import com.betsoft.casino.utils.TObject;
-import com.dgphoenix.casino.common.exception.CommonException;
-import com.dgphoenix.casino.common.util.Pair;
+import com.abs.casino.common.exception.CommonException;
+import com.abs.casino.common.util.Pair;
 import com.abs.casino.common.util.RNG;
 import com.dgphoenix.casino.common.util.string.StringUtils;
 import org.kynosarges.tektosyne.geometry.PointD;
diff --git a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/PlayerRoundInfo.java b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/PlayerRoundInfo.java
index a96edf2f8..245531dbf 100644
--- a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/PlayerRoundInfo.java
+++ b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/PlayerRoundInfo.java
@@ -7,7 +7,7 @@ import com.betsoft.casino.mp.bgdragonstone.model.math.MathData;
 import com.betsoft.casino.mp.model.GameType;
 import com.betsoft.casino.mp.model.IEnemyType;
 import com.betsoft.casino.mp.model.Money;
-import com.dgphoenix.casino.common.util.Pair;
+import com.abs.casino.common.util.Pair;
 import com.google.common.util.concurrent.AtomicDouble;
 import java.util.*;
 import java.util.concurrent.atomic.AtomicInteger;
diff --git a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/PrivateBTGWaitingGameState.java b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/PrivateBTGWaitingGameState.java
index c9afa0b00..8630fb78d 100644
--- a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/PrivateBTGWaitingGameState.java
+++ b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/PrivateBTGWaitingGameState.java
@@ -1,7 +1,7 @@
 package com.betsoft.casino.mp.bgdragonstone.model;
 
 import com.betsoft.casino.mp.AbstractPrivateBTGWaitingGameState;
-import com.dgphoenix.casino.common.exception.CommonException;
+import com.abs.casino.common.exception.CommonException;
 
 public class PrivateBTGWaitingGameState extends AbstractPrivateBTGWaitingGameState<GameRoom, Seat, GameMap, PrivateBTGWaitingGameState> {
     public PrivateBTGWaitingGameState() {}
diff --git a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/QualifyGameState.java b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/QualifyGameState.java
index 54c14e6ce..93c4dcee4 100644
--- a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/QualifyGameState.java
+++ b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/QualifyGameState.java
@@ -2,7 +2,7 @@ package com.betsoft.casino.mp.bgdragonstone.model;
 
 import com.betsoft.casino.mp.common.AbstractQualifyGameState;
 import com.betsoft.casino.mp.model.IGameState;
-import com.dgphoenix.casino.common.exception.CommonException;
+import com.abs.casino.common.exception.CommonException;
 
 public class QualifyGameState extends AbstractQualifyGameState<GameRoom, Seat, GameMap, QualifyGameState> {
 
diff --git a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/RoomSeats.java b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/RoomSeats.java
index bb9d97802..9a3550f16 100644
--- a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/RoomSeats.java
+++ b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/RoomSeats.java
@@ -1,6 +1,6 @@
 package com.betsoft.casino.mp.bgdragonstone.model;
 
-import com.dgphoenix.casino.common.exception.CommonException;
+import com.abs.casino.common.exception.CommonException;
 import java.io.Serializable;
 import java.util.Arrays;
 import java.util.Iterator;
diff --git a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/WaitingPlayersGameState.java b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/WaitingPlayersGameState.java
index ec264f38f..9eb6f099c 100644
--- a/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/WaitingPlayersGameState.java
+++ b/mp-server/games/bg_dragonstone/src/main/java/com/betsoft/casino/mp/bgdragonstone/model/WaitingPlayersGameState.java
@@ -1,7 +1,7 @@
 package com.betsoft.casino.mp.bgdragonstone.model;
 
 import com.betsoft.casino.mp.AbstractBattlegroundWaitingPlayersGameState;
-import com.dgphoenix.casino.common.exception.CommonException;
+import com.abs.casino.common.exception.CommonException;
 
 public class WaitingPlayersGameState extends AbstractBattlegroundWaitingPlayersGameState<GameRoom, Seat, GameMap, WaitingPlayersGameState> {
     public WaitingPlayersGameState() {}
diff --git a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/EnemyGame.java b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/EnemyGame.java
index 83a65d668..9dd284588 100644
--- a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/EnemyGame.java
+++ b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/EnemyGame.java
@@ -19,7 +19,7 @@ import com.betsoft.casino.mp.service.ISpawnConfigProvider;
 import com.betsoft.casino.mp.service.ITransportObjectsFactoryService;
 import com.betsoft.casino.teststand.TestStandFeature;
 import com.betsoft.casino.teststand.TestStandLocal;
-import com.dgphoenix.casino.common.exception.CommonException;
+import com.abs.casino.common.exception.CommonException;
 import com.abs.casino.common.util.RNG;
 import org.apache.logging.log4j.Logger;
 
diff --git a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/GameMap.java b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/GameMap.java
index 5178b729b..d65d08c66 100644
--- a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/GameMap.java
+++ b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/GameMap.java
@@ -10,7 +10,7 @@ import com.betsoft.casino.mp.dragonstone.model.math.config.SpawnConfig;
 import com.betsoft.casino.mp.model.*;
 import com.betsoft.casino.mp.model.movement.*;
 import com.betsoft.casino.mp.movement.SpiderSwarmAngleTrajectoryGenerator;
-import com.dgphoenix.casino.common.util.Pair;
+import com.abs.casino.common.util.Pair;
 import com.abs.casino.common.util.RNG;
 import com.esotericsoftware.kryo.Kryo;
 import com.esotericsoftware.kryo.io.Input;
diff --git a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/GameRoom.java b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/GameRoom.java
index 901e4517d..8e23431a8 100644
--- a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/GameRoom.java
+++ b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/GameRoom.java
@@ -12,7 +12,7 @@ import com.betsoft.casino.mp.model.movement.Trajectory;
 import com.betsoft.casino.mp.model.room.ISingleNodeRoomInfo;
 import com.betsoft.casino.mp.service.*;
 import com.betsoft.casino.mp.web.IGameSocketClient;
-import com.dgphoenix.casino.common.exception.CommonException;
+import com.abs.casino.common.exception.CommonException;
 import com.hazelcast.core.IExecutorService;
 import org.apache.logging.log4j.Logger;
 import org.springframework.context.ApplicationContext;
diff --git a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/PlayGameState.java b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/PlayGameState.java
index b1998fb08..58a66999b 100644
--- a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/PlayGameState.java
+++ b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/PlayGameState.java
@@ -12,8 +12,8 @@ import com.betsoft.casino.teststand.TestStandFeature;
 import com.betsoft.casino.teststand.TestStandLocal;
 import com.betsoft.casino.teststand.TeststandConst;
 import com.betsoft.casino.utils.TObject;
-import com.dgphoenix.casino.common.exception.CommonException;
-import com.dgphoenix.casino.common.util.Pair;
+import com.abs.casino.common.exception.CommonException;
+import com.abs.casino.common.util.Pair;
 import com.abs.casino.common.util.RNG;
 import com.dgphoenix.casino.common.util.string.StringUtils;
 import org.kynosarges.tektosyne.geometry.PointD;
diff --git a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/PlayerRoundInfo.java b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/PlayerRoundInfo.java
index 9e86c5c4c..beda8d9a4 100644
--- a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/PlayerRoundInfo.java
+++ b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/PlayerRoundInfo.java
@@ -7,7 +7,7 @@ import com.betsoft.casino.mp.dragonstone.model.math.MathData;
 import com.betsoft.casino.mp.model.GameType;
 import com.betsoft.casino.mp.model.IEnemyType;
 import com.betsoft.casino.mp.model.Money;
-import com.dgphoenix.casino.common.util.Pair;
+import com.abs.casino.common.util.Pair;
 import com.google.common.util.concurrent.AtomicDouble;
 
 import java.util.*;
diff --git a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/QualifyGameState.java b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/QualifyGameState.java
index 4dea7b0c8..1b9cbee00 100644
--- a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/QualifyGameState.java
+++ b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/QualifyGameState.java
@@ -1,7 +1,7 @@
 package com.betsoft.casino.mp.dragonstone.model;
 
 import com.betsoft.casino.mp.common.AbstractQualifyGameState;
-import com.dgphoenix.casino.common.exception.CommonException;
+import com.abs.casino.common.exception.CommonException;
 
 /**
  * User: flsh
diff --git a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/RoomSeats.java b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/RoomSeats.java
index 95fa002b8..303c09f70 100644
--- a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/RoomSeats.java
+++ b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/RoomSeats.java
@@ -1,6 +1,6 @@
 package com.betsoft.casino.mp.dragonstone.model;
 
-import com.dgphoenix.casino.common.exception.CommonException;
+import com.abs.casino.common.exception.CommonException;
 
 import java.io.Serializable;
 import java.util.Arrays;
diff --git a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/WaitingPlayersGameState.java b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/WaitingPlayersGameState.java
index 873465419..6b279bf0e 100644
--- a/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/WaitingPlayersGameState.java
+++ b/mp-server/games/dragonstone/src/main/java/com/betsoft/casino/mp/dragonstone/model/WaitingPlayersGameState.java
@@ -1,7 +1,7 @@
 package com.betsoft.casino.mp.dragonstone.model;
 
 import com.betsoft.casino.mp.common.AbstractSingleNodeWaitingPlayerGameState;
-import com.dgphoenix.casino.common.exception.CommonException;
+import com.abs.casino.common.exception.CommonException;
 
 /**
  * User: flsh
